File: Final_results/pedestrians/pedestriancase/pys/switcher.py
import matlab.engine
import numpy as np 
from timescalingex import timescalingex 
#calling MPC 
eng = matlab.engine.start_matlab()
eng.addpath(r'/home/raghu/Documents/switching_main/mtlb')
[stateRobo,stateObst,vlast,wlast,vsend,wsend]=eng.launch_iros(0,0,15,0,0,10,1,1)
eng.quit()

#storing output arguments
stateRobo=np.array(stateRobo)
stateObst=np.array(stateObst)
vsend = np.array(vsend)
wsend = np.array(wsend)
vl = float(vlast)
wl=float(wlast)
ends=0

#calling timescaling 
velocity,omega = timescalingex(ends,stateRobo,stateObst,vl,wl)

print('I came back! Things are good :)')

File: Final_results/roadbound/rb_no_obs_test/mtlb_prev/switcher.m
%The integration
    [stateRobo,stateObst,vlast,wlast,vsend,wsend]=launch_iros(x0,y0,x_ob,y_ob,theta_in,na,mela,ob);
    system('python3 /../time_scaling/timescalingex.py stateRobo stateObst vl wl');
    
   
    
    
    
    